examples/spack/Dockerfile

```diff
@@ -59,24 +62,16 @@ RUN spack compiler find --scope system
 # Test: Some basic commands.
 RUN which spack
 RUN spack --version
+RUN spack compiler find
 RUN spack compiler list
 RUN spack compiler list --scope=system
 RUN spack compiler list --scope=user
 RUN spack compilers
-RUN spack spec netcdf
+RUN spack spec charliecloud
 
 # Test: Install Charliecloud.
-RUN spack spec charliecloud+builder
-RUN spack install charliecloud+builder
```

Comment:
Did we not want to test installing `ch-grow` any more with Spack?

Reply:
The builder variant has been removed. You always get ch-grow now :).